### DISCUSSION TITLE
How Flink CDC uses SQL to filter out unnecessary transaction operations through the op field

### DISCUSSION THREAD

**Discussion by wqlsdb** [apache/flink-cdc#2525] [Q&A] [answered]
如下所示，Flink CDC 如何使用 SQL 通过 op 字段过滤掉不必要的事务操作！[a21383ba660413c323241253ac14409](https://github.com/ververica/flink-cdc-connectors/assets/84709143/3813a871-195d-4510-9a40- 7879cb10b588)

**Comment by wqlsdb** [comment]
![a21383ba660413c323241253ac14409](https://github.com/ververica/flink-cdc-connectors/assets/84709143/38880d17-2000-4240-b424-8dacd1d3722e)

**Comment by JimZyqq** [comment, marked as answer]
debezium.skipped.operations

**Reply by wqlsdb** [reply]
Thanks

**Comment by JimZyqq** [comment]
It is recommended to use the datastream API as the table API may result in data loss
